pkg/asset/machines/machineconfig/karg.go

```diff
@@ -0,0 +1,34 @@
+package machineconfig
+
+import (
+	"path/filepath"
+
+	igntypes "github.com/coreos/ignition/config/v2_2/types"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+
+	"github.com/openshift/installer/pkg/asset/ignition"
+)
+
+const (
+	kargDir = "/etc/default/rhcos/karg/"
```

Comment:
Should be `/etc/default/rhcos/kargs` ... where `kargs` is a file with requested changes to operate on.